Terminal.Gui/Core/MouseEventEventArgs.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Terminal.Gui {
+	/// <summary>
+	/// Specifies the event arguments for <see cref="Terminal.Gui.MouseEvent"/>. This is a higher-level construct
+	/// than the wrapped <see cref="MouseEvent"/> class and is used for the events defined on <see cref="View"/>
+	/// and subclasses of View (e.g. <see cref="View.MouseEnter"/> and <see cref="View.MouseClick"/>).
+	/// </summary>
+	public class MouseEventEventArgs : EventArgs {
```

Comment:
Looks like I misnamed this now as `MouseEventEventArgs` instead of `MouseEventArgs`

Reply:
Fix is in https://github.com/gui-cs/Terminal.Gui/pull/2435

Reply:
@tznind I did the change on purpose to `MouseEventEventArgs` because it's related with the `MouseEvent` class, for the same reason why there is a `MouseFlagsEventArgs` related with `MouseFlags` enum.